frontend/www/js/omegaup/common/index.ts

```diff
@@ -2,9 +2,28 @@ import Homepage from '../components/homepage/Homepage.vue';
 import { OmegaUp } from '../omegaup';
 import { types } from '../api_types';
 import Vue from 'vue';
+import * as ui from '../ui';
+import T from '../lang';
 
 OmegaUp.on('ready', () => {
   const payload = types.payloadParsers.IndexPayload();
+  const commonPayload = types.payloadParsers.CommonPayload();
+
+  if (commonPayload.pendingDaysForVerification != null) {
+    if (commonPayload.pendingDaysForVerification < 5) {
+      ui.warning(
+        ui.formatString(T.accountVerifyWarning, {
+          days: commonPayload.pendingDaysForVerification,
+        }),
+      );
+    } else if (commonPayload.pendingDaysForVerification <= 7) {
+      ui.error(
```

Comment:
and it should be a warning message:
```suggestion
      ui.warning(
```